docs/csharp/language-reference/tokens/interpolated.md

```diff
@@ -54,24 +53,24 @@ The following example shows how to include a brace in a result string and how to
 
 [!code-csharp-interactive[example with ternary conditional operator](~/samples/snippets/csharp/language-reference/tokens/string-interpolation.cs#3)]
 
-A verbatim interpolated string starts with the `$` character followed by the `@` character. For more information about verbatim strings, see the [string](../keywords/string.md) and [verbatim identifier](verbatim.md) topics.
+An interpolated verbatim string starts with the `$` character followed by the `@` character. For more information about verbatim strings, see the [string](../keywords/string.md) and [verbatim identifier](verbatim.md) topics.
```

Comment:
@BillWagner I've swapped the "verbatim" and "interpolated" terms to match the order in the text of the CS8401 compile error:
>To use '@$' instead of '$@' for an interpolated verbatim string, please use language version 'preview' or greater.